evm/scripts/benchgethprecompiles.py

```diff
@@ -138,20 +138,7 @@ def saveResults(precompile_benchmarks):
         for test_result in precompile_benchmarks:
             writer.writerow({"test_name" : test_result['name'], "gas" : test_result['gas'], "time" : test_result['time']})
 
-
-def apply_patch_for_ecpairing_input():
-    git_cmd = shlex.split("git apply /scripts/geth_ecpairing_precompile_rollup_input.patch")
```

Comment:
If this file is no longer referenced, it should be removed.

Btw is it not needed anymore? Is it part of ewasm-benchmarking/go-ethereum?

Reply:
I updated https://github.com/ewasm-benchmarking/go-ethereum/tree/v1.9.14-benchmarking and removed the patchfile from this PR.